src/ariel-os-storage/src/lib.rs

```diff
@@ -135,6 +135,11 @@ pub async fn remove(key: &str) -> Result<(), sequential_storage::Error<FlashErro
     lock().await.remove(key).await
 }
 
+/// Resets the flash in the entire flash range.
+pub async fn erase_all() -> Result<(), sequential_storage::Error<FlashError>> {
+    lock().await.erase_all().await
```

Comment:
You'll need to also write the marker, as done [here](https://github.com/ariel-os/ariel-os/pull/621/files#diff-944be64661985335476ae5baf50db209678bda1c50ddbd71b38632fa2d6ed8d6R96). Otherwise, on next boot, everything will be erased again by `init()`.